maintainers/maintainer-list.nix

```diff
@@ -11461,6 +11461,12 @@
     githubId = 543055;
     name = "Shadaj Laddad";
   };
+  shadowrz = {
+    email = "23130178+ShadowRZ@users.noreply.github.com";
```

Comment:
Please use a none github email.

Reply:
There are other maintainers that uses a email like this. Actually the commit https://github.com/NixOS/nixpkgs/commit/64eb5ec86305e6547d5a7a37efda936584fbbafc uses such email address.

Also:
```console
$ sed -ne '/users.noreply.github.com/p' maintainers/maintainer-list.nix | wc -l
59
```

Anyway if this is undesired I'll change that.

Reply:
Changed.